src/DocBlock/Annotation.php

```diff
@@ -187,6 +168,33 @@ public function getTag()
         return $this->tag;
     }
 
+    /**
+     * @return TypeExpression
+     *
+     * @internal until 2.19
```

Comment:
```suggestion
     * @internal
```

to speed up review. let's separately discuss if we would like to make it public